cscs-checks/microbenchmarks/dgemm/dgemm.py

```diff
@@ -14,8 +14,17 @@ def __init__(self):
         # the perf patterns are automaticaly generated inside sanity
         self.perf_patterns = {}
 
-        self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc']
-        self.valid_prog_environs = ['PrgEnv-gnu', 'PrgEnv-intel']
+        self.valid_systems = [
+            'daint:gpu', 'daint:mc',
+            'dom:gpu', 'dom:mc',
+            'kesch:cn', 'kesch:pn'
+        ]
+
+        if self.current_system.name == ['daint', 'dom']:
```

Comment:
This will never be true! You should write it as follows:

```suggestion
        if self.current_system.name in ['daint', 'dom']:
```